tower-grpc-build/src/client.rs

```diff
@@ -52,6 +52,11 @@ impl ServiceGenerator {
             .generic("T")
             .derive("Debug")
             .derive("Clone")
+            .doc(&service.comments.leading
```

Comment:
I _think_ using only the `leading` field on `service.comments` here (and elsewhere in this branch) is probably the right thing, as I'm guessing the `leading_detached` comments (which are separated from the service declaration by one or more newlines) are not typically describing the API of the service definition. 

However, if anyone else thinks we ought to include both sets of comments (@ebkalderon?) we can always add them in a subsequent PR!